.github/workflows/continuous_integration.yml

```diff
@@ -78,6 +83,11 @@ jobs:
       with:
         dotnet-version: '6.0.x'
 
+    - name: Setup .NET Core SDK 7.0.x
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '7.0.x'
+
```

Comment:
Although not wrong, but we should move to the more condensed version supported by `setup-dotnet@v2`:
```
    - name: Setup .NET
      uses: actions/setup-dotnet@v2
      with:
        dotnet-version: |
          3.1.x
          5.0.x,
          6.0.x,
          7.0.x
```